Modules/TPC/src/CheckOfTrendings.cxx

```diff
@@ -29,41 +30,127 @@
 namespace o2::quality_control_modules::tpc
 {
 
-void CheckOfTrendings::configure() {}
+void CheckOfTrendings::configure()
+{
+  if (auto param = mCustomParameters.find("chooseCheckMeanOrExpectedPhysicsValueOrBoth"); param != mCustomParameters.end()) {
+    checkChoice = param->second.c_str();
+    if (checkChoice != "mean" && checkChoice != "ExpextedPhysicsValue" && checkChoice != "both") {
```

Comment:
There are multiple instances of misspelt **ExpextedPhysicsValue** in this file. I think it would lead to problems as the setting in the json file would not be taken into account.